hardhat.config.ts

```diff
@@ -52,7 +58,7 @@ if (process.env.GOERLI_ETH_NODE_URL) {
   config.networks['goerli'] = {
     url: process.env.GOERLI_ETH_NODE_URL,
     accounts: [`0x${process.env.GOERLI_OWNER_PRIVATE_KEY}`],
-    gasPrice: 98000000000
+    gasPrice: 320000000000
```

Comment:
i think that we should have a way to overwrite it via env and make the default smaller (320 gwei is very high).

Reply:
ok